src/library/webService.py

```diff
@@ -12,13 +12,20 @@
                     datefmt='%d-%b-%y %H:%M:%S')
 
 repoFactory = RepoFactory(app)
-testRepo = repoFactory.create_repository("mysql", URI)
+webService = repoFactory.create_repository(URI)
```

Comment:
Change to `testRepo` since you are creating repo which is inside web service, not web service itself

Reply:
Changed to repository ( we are not testing anything here )